packages/native/src/components/date/DateRange.js

```diff
@@ -193,9 +193,10 @@ class DateRange extends Component {
 			this.setState({
 				currentDate,
 			}, () => {
+				this.updateQuery(value, props);
+				if (props.onValueChange) props.onValueChange(value);
```

Comment:
Not completely sure about this one.

Reply:
It seems correct. The pattern should always be:

1. state update
2. then, call updateQuery
3. call onValueChange